### DISCUSSION TITLE
Fetching wrong show with same name

### DISCUSSION THREAD

**Discussion by mash2k3** [itsToggle/plex_debrid#296] [General]
There is a 2021 show called Kung Fu, it got the right ones for season 1 and 2 but season 3 which is currently ongoing it keeps fetching the one for the 1972 show with same name.
Im sure this can be an issue for shows like The Office, Shameless Magnum PI etc...
Any body have a solution for this and also is there a way for the search to include year?

**Comment by itsToggle** [comment]
Hi, since a lot of show releases don't have a year in their title, plex_debrid is not scraping for the show name + year.

Here is what you could do:
Create a "character replacement rule" (/Settings/Scraper Settings/character replacement rules/) to replace the query that Plex_debrid is scraping for (`kung.fu`) with this (`kung.fu.2021`)


Here is what I could do:
If a release does mention the year, I could make sure that it's the correct one. If it doesnt mention the year, I could let plex_debrid download it anyway.

**Reply by mash2k3** [reply]
Do you know where in your roadmap updating single episodes will fall under?
For example updating an existing show on watchlist or overseerr request when a new episode airs.

**Reply by itsToggle** [reply]
That is already happening, new episodes are downloaded automatically for any watchlisted show

**Reply by mash2k3** [reply]
hmm, it is not for me atleast, does it also download new episodes for requested shows on jellyseerr?

**Reply by itsToggle** [reply]
Yeah. New episodes from shows requested through overseerr are downloaded as well.

**Reply by mash2k3** [reply]
thank you i will monitor it for the next few days
